packages/angular_devkit/build_angular/src/babel/presets/application.ts

```diff
@@ -103,6 +105,12 @@ export default function (api: unknown, options: ApplicationPresetOptions) {
   const plugins = [];
   let needRuntimeTransform = false;
 
+  if (options.angularLinker) {
+    // Babel currently is synchronous so import cannot be used
+    const { createEs2015LinkerPlugin } = require('@angular/compiler-cli/linker/babel');
+    plugins.push(createEs2015LinkerPlugin());
```

Comment:
For full external template source-map support we will need to provide a `FileSystem` and a `Logger` to this function. See https://github.com/angular/angular/pull/40237

Reply:
Logger is now integrated.  The filesystem is a little more complicated to integrate.  Once the upstream PR is in place, we can do a followup PR to add full sourcemap support to the tooling integration.